src/Nlp.cpp

```diff
@@ -28,26 +28,25 @@ NlpFstLoader::NlpFstLoader(std::vector<RawNlpRecord> &records, Json::Value norma
   std::string last_label;
   bool firstTk = true;
 
+  auto logger = logger::GetOrCreateLogger("NlpFstLoader");
   // fuse multiple rows that have the same id/label into one entry only
   for (auto &row : records) {
-    mNlpRows.push_back(row);
     auto curr_tk = row.token;
     auto curr_label = row.best_label;
     auto curr_label_id = row.best_label_id;
     auto punctuation = row.punctuation;
     auto curr_row_tags = row.wer_tags;
 
     // Update wer tags in records to real string labels
-    vector<string> real_wer_tags;
     for (auto &tag : curr_row_tags) {
-      auto real_tag = tag;
       if (mWerSidecar != Json::nullValue) {
-        real_tag = "###" + real_tag + "_" + mWerSidecar[real_tag]["entity_type"].asString() + "###";
+        tag.entity_type = mWerSidecar[tag.tag_id]["entity_type"].asString();
```

Comment:
what happened to the `###` stuff

Reply:
It's been delayed to the `WriteSbs` function https://github.com/revdotcom/fstalign/pull/55/files#diff-63af1040726873c90ede631a5381b72f24d13087ac1ecb3587bf151675b4e871R555
The goal of moving the wer tag to a new struct is to allow the intended formatting for SBS, but still make it easy to output just the tag ID to the NLP output